backend/src/main/resources/static/aas-registry-openapi.yaml

```diff
@@ -788,12 +788,12 @@ components:
             minLength: 1
             maxLength: 200
           # not supported for Catena-X
-          # subjectId:
-            # $ref: '#/components/schemas/Reference'
           value:
             type: string
             minLength: 1
             maxLength: 200
+          externalSubjectId:
```

Comment:
Does it make sense to further restrict what values can be inserted here? Do we know more about the structure of a BPN (e.g. lenght)? I know that we want to keep the tenant information independently of any specifics, but maybe it makes sense to make it configurable.